docs/distributions/ubuntu/faq.md

```diff
@@ -0,0 +1,52 @@
+# Why does my MacBook turn off in the middle of the Ubuntu installation?
+
+This seems to happen with certain MacBooks because the GRUB bootloader installer tries accessing the efivars/nvram, which Apple doesn't allow and the installer doesn't know what to do.
+
+There is a way to stop this. Boot into the installation media with an External USB Keyboard plugged in. Press e when you selected the "Try Ubuntu without installing" option.
+
+Scroll to the bottom with the arrow keys, and type in ``efi=noruntime``. With the External keyboard, then press CTRL+X or F10 to boot into the Live Media. The installation should work fine now.
+
+This issue has occured for anyone on the 16,1 and maybe the 16,4.
+
+(Credits to Redecorating#0350 for this fix)
+
+# Making the GRUB Menu appear
+
+The GRUB bootloader by default turns off the GRUB Menu. This means you can't boot into Bootcamp Windows if it's installed. This can be easily fixed after Ubuntu is fully installed.
+
+In a Terminal in Ubuntu, type in ``sudo nano /etc/default/grub``. Change line ``GRUB_TIMEOUT_STYLE`` to ``GRUB_TIMEOUT_STYLE=MENU``. Save the file by doing CTRL+X, Y, then enter.
```

Comment:
Even tho its nice to have documentation tell you exactly what to do, I'd expect someone who wants to install linux on their mac to be knowledgeable enough about linux to edit a file in their preferred editor.
In other wiki pages "edit file /path/to/file" is used instead of "type (nano / vim / whatever) /path/to/file)", also to keep it independent from the packages installed on a machine